test/sagemaker_tests/pytorch/training/conftest.py

```diff
@@ -500,6 +500,23 @@ def skip_dgl_test(
         pytest.skip(f"DGL binary is removed, skipping test")
 
 
+@pytest.fixture(autouse=True)
+def skip_pytorchddp_test(
+    request,
+    processor,
+    ecr_image,
+):
+    """Start from PyTorch 2.0.1 framework, SMDDP binary releases are decoupled from DLC releases.
+    For each currency release, Once SMDDP binary is added, we skip pytorchddp tests due to `pytorchddp` and `smdistributed` launcher consolidation.
+    See https://github.com/aws/sagemaker-python-sdk/pull/4698.
+    """
+    skip_dict = {">=2.1,<2.4": ["cu121"]}
```

Comment:
will skip condition be needed for pytorch version beyond 2.4?

Reply:
yes it will, for now it will have to be manually updated each currency when smddp is added until we have a script to detect whether smddp is installed or not.

I pin it to before 2.4 so that when currency come, I won't have to edit the skip condition.

Reply:
i see, thanks.